File: KSW/Python_daily/20240702/20240702.py
def solution(n):
    answer = 0
    sum_num = 0
    n = str(n)
    for num in n:
        sum_num += int(num)
    answer = sum_num


    return answer

# 무조건 쓰기전에 선언은 해야한다. 

File: KSW/JAVA_daily/2407/20240702/20240702.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        
        int[] pyArray = {0,0};
        char[] charArray = s.toCharArray();
        for(int i = 0; i < charArray.length; i++){
            if (charArray[i] == 'p' | charArray[i] == 'P'){
                pyArray[0] += 1;
            }
            else if (charArray[i] == 'y' | charArray[i] == 'Y'){
                pyArray[1] += 1;
            }
        }
        if (pyArray[0] == pyArray[1]){
            answer = true;
        }
        else {
            answer = false;
        }
        return answer;
    }
}
// java array 선언하는 것도 익숙해지자, 자료구조의 시작이다.